.gitlab-ci.yml

```diff
@@ -84,29 +84,32 @@ Prepare-rhel-internal:
   stage: prepare-rhel-internal
   extends: .terraform
   rules:
-    - if: '$CI_PIPELINE_SOURCE == "schedule" && $NIGHTLY == "true"'
+    - if: '$CI_PIPELINE_SOURCE == "schedule" && $RUNNER =~ /[\S]+rhel-9.0-[^ga][\S]+/ && $NIGHTLY == "true" && $VERSION_ID == "9.0"'
+    - if: '$CI_PIPELINE_SOURCE == "schedule" && $RUNNER =~ /[\S]+rhel-8.6-[^ga][\S]+/ && $NIGHTLY == "true" && $VERSION_ID == "8.6"'
   script:
     - schutzbot/prepare-rhel-internal.sh
   artifacts:
     paths:
-      - rhel-8.json
-      - rhel8internal.repo
-      - rhel-8-beta.json
+      - rhel-${VERSION_ID}.json
+      - rhel${VERSION_ID}internal.repo
```

Comment:
The original `rhel8internal.repo` name is also used in all of config.json for the runners:
https://github.com/osbuild/gitlab-ci-terraform/blob/main/aws/rhel-8.6-nightly-x86_64/config.json

to make sure that the override logic afterwards will update this file accordingly and you don't end up pulling RPMs from a different repository than intended, see https://github.com/osbuild/gitlab-ci-terraform/pull/6 and deploy.sh, line 112.

The change in name breaks this behavior and you can potentially pull RPMs from a different repo/compose, which are a different version and not what you wanted to test. The risk is slim but not zero.

Reply:
Thanks, I didn't notice that it was being used elsewhere. I guess it worked because during `prepare-rhel-internal.sh` the created repo gets `priority=1` so it would always be preferred over the others. I changed it to use just the major version in the filename to be sure though. 

This should be fine until we get to a state where we'll be testing two $major.$minor nightlies (eg. 8.6 and 8.7) but we can worry about that later, I'd prefer to have this PR resolved first.